requirements/tools.txt

```diff
@@ -5,18 +5,14 @@ autoflake==0.7
 autopep8==1.3.1
 Babel==2.4.0
 clint==0.5.1
-Django==1.11.1
 docformatter==0.8
 docutils==0.13.1
-Faker==0.7.11
 flake8==3.3.0
-flaky==3.3.0
 imagesize==0.7.1
 isort==4.2.5
 Jinja2==2.9.6
 MarkupSafe==1.0
 mccabe==0.6.1
-numpy==1.12.1
```

Comment:
Not sure. I didn't check in detail. I think pytz is there as a sphinx dependency as well as on its own merits.

The main ones I wanted to get rid of were numpy and Django, which are relatively slow to install.